src/v/finjector/stress_fiber.cc

```diff
@@ -17,11 +17,15 @@
 #include <seastar/core/abort_source.hh>
 #include <seastar/core/coroutine.hh>
 #include <seastar/core/gate.hh>
+#include <seastar/core/scheduling.hh>
 #include <seastar/core/timer.hh>
+#include <seastar/core/with_scheduling_group.hh>
+#include <seastar/coroutine/maybe_yield.hh>
 
 class stress_payload {
 public:
-    // Starts the stress payload with the given configuration.
+    // Starts the stress payload with the given configuration in the given
```

Comment:
nit: would be great if this could be pulled out into a separate commit. You never know when it's useful to cherry-pick something, and it'd be nice to not have to pull in the wider maybe_yield change if we do (though tbf the scheduling group change is small)

Reply:
Yes good point, done